### PR TITLE
Fixed German Translation

### PR DESCRIPTION
**Pull request by Baumfinder**
Name of Project was falsely translated

### CODE REVIEW

**Review by MarcinOrlowski — APPROVED** [review]

**Comment by MarcinOrlowski** [comment]
I noticed there's another occurrence of `Logisim-Entwicklung` in project files (but not code). Can you fix that too?

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
I noticed there's another occurrence of `Logisim-Entwicklung` in project files (in `src/main/resources/doc/de/html/guide/subcirc/sub-appear.html`). Can you update this PR and fix that too?

**Comment by Baumfinder** [comment]
@MarcinOrlowski  Thanks for telling me, I've fixed that too.

File: src/main/resources/doc/de/html/guide/subcirc/sub-appear.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h1>
         Auswählen des Erscheinungsbildes eines Subcircuits
       </h1>
       <p>
-        Es gibt vier Modelle für die grafische Darstellung der Schaltung (Logisim-Entwicklung,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert). Die Version der Präsentation ist in der Eigenschaft Circuit Appearance auswählbar. 
+        Es gibt vier Modelle für die grafische Darstellung der Schaltung (Logisim-evolution,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert). Die Version der Präsentation ist in der Eigenschaft Circuit Appearance auswählbar.
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/de/html/guide/subcirc/sub-appear.html:20]: May I also request adding single space after each `,` in `(Logisim-evolution,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert)` line?

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

**Review by MarcinOrlowski — APPROVED** [review]

**Comment by MarcinOrlowski** [comment]
Thanks for correcting these issues.

**Pull request merged**